docs/news.rst

```diff
@@ -1,26 +1,38 @@
-
-
-GSOC 2017
----------
-
-.. image:: https://developers.google.com/open-source/gsoc/resources/downloads/GSoC-logo-horizontal.svg
-  :target: https://summerofcode.withgoogle.com/organizations/4793296782098432/
-  :width: 458 px
-
-TARDIS participates in the Google Summer of Code 2017 program as part of the Python Software Foundation!
-
-
-SOCIS 2016
-----------
-
-TARDIS is part of ESA's Summer of Code in Space 2016 program!
-
-GSOC 2016
----------
-
-.. image:: https://developers.google.com/open-source/gsoc/resources/downloads/GSoC-logo-horizontal.svg
-  :target: https://summerofcode.withgoogle.com/organizations/4793296782098432/
-  :width: 458 px
-
-TARDIS has been selected as a mentoring organisation for the Google Summer of Code 2016 program!
-
+.. raw:: html
+
+    <div class = "twitter-block"  style = "width:100%; display: flex; justify-content: center;">
+    <a class="twitter-timeline" data-width="850" data-height="900" href="https://twitter.com/tardis_sn?ref_src=twsrc%5Etfw">Tweets by tardis_sn</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+    </div>
+
+
+
+    <script>
+         jQuery('.twitter-block').delegate('#twitter-widget-0', 'DOMSubtreeModified propertychange', function() {
+        // function call to override the base twitter styles
+        customizeTweetMedia();
+        });
+
+        var customizeTweetMedia = function() {
+        jQuery('.twitter-block').find('.twitter-timeline').contents().find('.timeline-Header-title.u-inlineBlock').css('font-size', '18px');
+        jQuery('.twitter-block').find('.twitter-timeline').contents().find('.timeline-Header-title.u-inlineBlock').children().css('font-size', '18px');
+        jQuery('.twitter-block').find('.twitter-timeline').contents().find('.timeline-Tweet-text').css({
+            'font-size': '16px',
+            'font': 'Lato,proxima-nova,Helvetica Neue,Arial,sans-serif;',
```

Comment:
It will be `'font-family': 'Lato,proxima-nova,Helvetica Neue,Arial,sans-serif',` instead - that's why fonts didn't appear to be changed.